appui/src/pages/videolink/VideoLink.js

```diff
@@ -0,0 +1,45 @@
+import './VideoLink.scss';
+import React from 'react';
+import ReactPlayer from 'react-player';
+
+function VideoLink() {
+    return (
+        <div id="videolink">
+            <div className="flex-container-column">
+                <h1 className="heading">Note Together</h1>
+            </div>
+            <div className="grid">
+                <div className="flex-container-column">
+                    <div className="label">
+                        Video Title 
+                    </div>
+                    <div className="video-box">
+                        <ReactPlayer className = "video-player" url='https://www.youtube.com/watch?v=dQw4w9WgXcQ' controls={true}/>
```

Comment:
Might be worth adding our own component to wrap this to allow us to later add onto the player